test/knex-dev-practice.test.js

```diff
@@ -0,0 +1,15 @@
+const connection = require('../knexfile');
```

Comment:
These were great exploratory tests while learning knex, but I don't think we want to keep them around